include/chibi/sexp.h

```diff
@@ -197,15 +197,20 @@ enum sexp_types {
 #define SEXP_STRING_CURSOR SEXP_FIXNUM
 #endif
 
-/* procedure flags */
-#define SEXP_PROC_NONE 0uL
-#define SEXP_PROC_VARIADIC 1uL
-#define SEXP_PROC_UNUSED_REST 2uL
-
 #ifdef _WIN32
+#if defined(_MSC_VER) && SEXP_64_BIT
+/* On SEXP_64_BIT, 128bits arithmetic is mandatory */
+#error Unsupported configuration
+#endif
+#ifdef SEXP_64_BIT
```

Comment:
`SEXP_64_BIT` shouldn't be evaluated with `#ifdef`. Addressed with https://github.com/ashinn/chibi-scheme/pull/438/commits/431bc38976ca39ed083fd0334dff9f1cd30d1d96